Working with forms (#2735)
umbraco/UmbracoDocs: Umbraco-Uno/Working-With-Forms/How-to-check-your-replies-from-forms/index.md

---
versionFrom: 8.0.0
---

> # How to Check your form replies

Reviewer: ```suggestion
# How to check your form replies
```

> Suppose you have a form on your frontend, and you want to check if you go some replies to your form. However, you do not know how to see your replies. Then this article is for you.

Reviewer: ```suggestion
Suppose you have a form on your frontend, and you want to check if you got some replies to your form. In this article, you will learn how you can find and view these replies.
```

> What you will want to do is, go to the forms section in the backoffice, then in the big window located in the middle of the screen. Depending on how many forms you have, this can look differently. If you only have one form, there will be a box showing you how many replies you have. Clicking this box will bring you to an overview of all the replies, and you can click into each of them to see them filled out.

Reviewer: How about splitting this into bullets instead? 🤔 

```suggestion
What you will want to do is, go to the forms section in the backoffice, then in the big window located in the middle of the screen. Depending on how many forms you have, this can look differently. If you only have one form, there will be a box showing you how many replies you have. Clicking this box will bring you to an overview of all the replies, and you can click into each of them to see them filled out.
```
